Dockerfile

```diff
@@ -77,4 +77,7 @@ ONBUILD COPY src/Pyz $WORKDIR/src/Pyz
 ONBUILD COPY config $WORKDIR/config
 ONBUILD COPY public $WORKDIR/public
 ONBUILD RUN /entrypoint.sh build-shop
+ONBUILD COPY codeception.yml $WORKDIR/
+ONBUILD COPY setup_test $WORKDIR/
```

Comment:
Oh right. I actually tried that, but in fact it doesn't work the way we want, because I want to copy the file _setup_test_ and the folder _tests/_, and it copies the content of the tests folder to the destination, not the folder itself. This messes up with everything.